marketplacetests/marketplace/regions/add_review.py

```diff
@@ -3,22 +3,24 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from marionette.by import By
-from gaiatest.apps.base import Base
 
+from marketplacetests.marketplace.app import Marketplace
+from marketplacetests.marketplace.regions.app_details import Details
 
-class AddReview(Base):
+class AddReview(Marketplace):
     """
     Page for adding reviews.
     """
 
+    _page_loaded_locator = (By.CSS_SELECTOR, 'form.add-review-form')
+
     _add_review_input_field_locator = (By.ID, 'review-body')
     _submit_review_button_locator = (By.CSS_SELECTOR, 'button[type="submit"]')
-    _review_box_locator = (By.CSS_SELECTOR, '.add-review-form')
     _rating_locator = (By.CSS_SELECTOR, ".ratingwidget.stars-0 > label[data-stars='%s']")
 
     def __init__(self, marionette):
-        Base.__init__(self, marionette)
-        self.wait_for_element_displayed(*self._review_box_locator)
+        Marketplace.__init__(self, marionette)
+        self.wait_for_page_loaded()
```

Comment:
Could we move this into the base `__init__` to avoid having to override it in all subclasses?